### PR TITLE
Misc updates for consistency with scikit-image 0.20

### PR DESCRIPTION
**Pull request by grlee77**
related to #419

The larger feature updates related to skimage 0.20 were opened in separate PRs. This one contains any other minor bug fixes and documentation tweaks.

### CODE REVIEW

File: python/cucim/src/cucim/skimage/restoration/_denoise.py [hunk 1]
```diff
@@ -97,61 +97,84 @@ def _denoise_tv_chambolle_nd(image, weight=0.1, eps=2.0e-4, max_num_iter=200):
...
         Otherwise, this parameter indicates which axis of the array corresponds
         to channels.
 
+        .. versionadded:: 0.19
+           ``channel_axis`` was added in 0.19.
+
     Returns
     -------
-    out : ndarray
+    u : cndarray
```
Comment (gigony) [inline python/cucim/src/cucim/skimage/restoration/_denoise.py:152]: `cndarray` might be a typo?

**Review by gigony — APPROVED** [review]
Looks good to me! Thanks!

File: python/cucim/src/cucim/skimage/restoration/_denoise.py (hunk 1 above)
Comment (grlee77) [reply python/cucim/src/cucim/skimage/restoration/_denoise.py:152]: Yeah, this should be either `ndarray` or `cupy.ndarray`

Comment (grlee77) [reply python/cucim/src/cucim/skimage/restoration/_denoise.py]: changed to `ndarray`

**Comment by grlee77** [comment]
@gpucibot merge

**Pull request merged**